src/migrations/00054-migrate-release-versioning-fields.sql

```diff
@@ -0,0 +1,105 @@
+UPDATE "release"
+SET "revision" = NULL
+WHERE "release type" = 'draft'
```

Comment:
Do we need a further migration for the draft releases as well to set `revision` to timestamp?

Reply:
No, draft releases will have a NULL `revision`, which actually will be the source of truth on whether a release is draft or not.
The timestamp part will only be in the definition of the `raw` part of the `version` computed term.

I hope I didn't misunderstood your question though. So, does that make sense?

Reply:
> The timestamp part will only be in the definition of the `raw` part of the `version` computed term.

This answered it for me Thodori :+1: